Core/Core.php

```diff
@@ -762,14 +761,14 @@ function create_function ($args, $code) {}
  * @param resource $resource <p>
  * The evaluated resource handle.
  * </p>
- * @return string If the given <i>handle</i> is a resource, this function
+ * @return string|false If the given <i>handle</i> is a resource, this function
```

Comment:
Hi @voku ,
Could you please clarify about this change? As I see this function always return string https://github.com/php/php-src/blob/master/Zend/zend_builtin_functions.c#L1404